src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -348,7 +348,7 @@ KeyChord ConversionTrait<KeyChord>::FromJson(const Json::Value& json)
         {
             throw winrt::hresult_invalid_argument{};
         }
-        return _fromString(til::u8u16(keyChordText));
+        return keyChordText.empty() ? nullptr : _fromString(til::u8u16(keyChordText));

```

Comment:
I think it would be better to move this check into `_fromString` so that the above `KeyChordSerialization::FromString` method also doesn't explode.